content/docs/releases/README.md

```diff
@@ -222,15 +223,15 @@ newer Kubernetes features.
 The table below lists the major Kubernetes distributions we check. In parentheses next to each release is the <abbr title="End-of-life">EOL</abbr>
 for that release. EOL dates often change throughout the lifecycle of a release.
 
-The "Oldest Kubernetes Release" is the oldest release we deemed relevant to the next cert-manager release, as of 2024-09-25
+The "Oldest Kubernetes Release" is the oldest release we deemed relevant to the next cert-manager release, as of 2025-06-10
 
-|      Vendor           |  Oldest K8s Release   |                      Other Kubernetes Releases                       |
-|:---------------------:|:---------------------:|----------------------------------------------------------------------|
-|    [EKS][eks]         | 1.30 (Jul 2025)       | 1.31 (Nov 2025), 1.32 (TBD)                                          |
-|    [GKE][gke]         | 1.30 (Jul 2025)       | 1.31 (Sep 2025)                                                      |
-|    [AKS][aks]         | 1.30 (Jul 2025)       | 1.31 (Nov 2025), 1.32 (Mar 2026)                                     |
-| [OpenShift 4][os]     | 1.28 (4.15, Aug 2025) | 1.29 (4.16, Dec 2025), 1.30 (4.17, Apr 2026)                         |
-| [OpenShift 4 EUS][os] | 1.27 (4.14, Oct 2025) | 1.29 (4.16, Jun 2026)                                                |
+| Vendor                | Oldest K8s Release    | Other Kubernetes Releases                                           |
+|:---------------------:|:---------------------:|---------------------------------------------------------------------|
+| [EKS][eks]            | 1.30 (Jul 2025)       | 1.31 (Nov 2025), 1.32 (Mar 2026), 1.33 (Jul 2026)                   |
+| [GKE][gke]            | 1.30 (Jul 2025)       | 1.31 (Jan 2026), 1.32 (Apr 2026)                                    |
+| [AKS][aks]            | 1.30 (Jul 2025)       | 1.31 (Nov 2025), 1.32 (Mar 2026), 1.33 (May 2026)                   |
+| [OpenShift 4][os]     | 1.28 (4.15, Aug 2025) | 1.29 (4.16, Dec 2025), 1.30 (4.17, Apr 2026), 1.31 (4.18, Aug 2026) |
+| [OpenShift 4 EUS][os] | 1.27 (4.14, Oct 2025) | 1.29 (4.16, Jun 2026), 1.31 (4.18, Feb 2027)                        |
```

Comment:
I verified these updated EOL dates by visting the links:
 * https://endoflife.date/amazon-eks
 * https://endoflife.date/google-kubernetes-engine
 * https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#aks-kubernetes-release-calendar
 * https://endoflife.date/red-hat-openshift